libretroshare/src/util/rsnet.cc

```diff
@@ -174,3 +485,18 @@ std::string rs_inet_ntoa(struct in_addr in)
 	rs_sprintf(str, "%u.%u.%u.%u", (int) bytes[0], (int) bytes[1], (int) bytes[2], (int) bytes[3]);
 	return str;
 }
+
+std::string rs_inet_ntoa(struct in6_addr& in)
```

Comment:
I think we already have an quivalent function with sockaddr_storage, plus looks like the param should be const here

Reply:
This one?
https://man7.org/linux/man-pages/man3/inet_ntop.3.html